packages/progress-indicator/test/lion-progress-indicator.test.js

```diff
@@ -1,23 +1,125 @@
+      expect(el.indeterminate).to.be.true;
     });
+  });
 
-    it('sets the right role', async () => {
-      const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
-      expect(el.getAttribute('role')).to.equal('status');
+  describe('determinate', async () => {
+    it('is determinate when has value attribute', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="25" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.indeterminate).to.be.false;
     });
 
-    it('sets aria-live to "polite"', async () => {
-      const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
-      expect(el.getAttribute('aria-live')).to.equal('polite');
+    it('can update value', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="25" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuenow')).to.be.eq('25');
+      el.value = 30;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuenow')).to.be.eq('30');
+    });
+
+    it('can update min', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="50" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuemin')).to.be.eq('0');
+      el.min = 30;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuemin')).to.be.eq('30');
+    });
+
+    it('can update max', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="50" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuemax')).to.be.eq('100');
+      el.max = 70;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuemax')).to.be.eq('70');
+    });
+
+    it('min & max limits value', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="150" aria-label="foo"></lion-progress-indicator> `,
+      );
+      // sets to default max: 100
+      expect(el.getAttribute('aria-valuenow')).to.be.eq('100');
+      el.value = -20;
+      await el.updateComplete;
+      // sets to default min: 0
+      expect(el.getAttribute('aria-valuenow')).to.be.eq('0');
+    });
+
+    it('has a _progressPercentage getter to get the progress percentage', async () => {
+      const el = await fixture(
+        html`
+          <lion-progress-indicator max="50" value="10" aria-label="foo"></lion-progress-indicator>
+        `,
+      );
+      expect(el._progressPercentage).to.be.eq(20);
+    });
+  });
+
+  describe('Accessibility', () => {
+    describe('indeterminate', () => {
+      it('passes a11y test', async () => {
+        const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
+        await expect(el).to.be.accessible();
+      });
+
+      it('by default', async () => {
```

Comment:
sets the progressbar role? (and we should actually also allow to override it, but could be a later action point for the whole codebase)